static/index.html

```diff
@@ -398,7 +400,11 @@ <h3 class="header mb-2">Add panel to dashboard</h3>
     <script src="./js/dashboard-from-logs.js?cb=1_1_10"></script>
     <script src="./js/query-builder.js"></script>
     <script src="./js/lib/echarts.min.js"></script>
+    {{ .IndexExtraImports | safeHTML }}
+
+    {{ .RunRbacAuthCheck | safeHTML }}
```

Comment:
we should add this check to all html files. Before loading , check for valid session/token